lib/tasks/config.rake

```diff
@@ -0,0 +1,111 @@
+
+namespace :config do
+  
+  role_list = %w(automate database_owner ems_metrics_coordinator ems_metrics_collector ems_metrics_processor database_operations event git_owner notifier ems_inventory ems_operations rhn_mirror reporting scheduler smartproxy smartstate user_interface web_services websocket)
+  desc "Usage information regarding available tasks"
+  task :usage do
+    puts "The following configuration tasks are available, arguments between [] are optional:"
+    puts " List all roles available           - Usage: rake config:list_roles"
+    puts " List appliance active roles        - Usage: rake config:list_active_roles [APPLIANCE_ID=appliance_id]"
+    puts " Set appliance roles                - Usage: rake config:set_roles SERVER_ROLES='[\"roles\", \"json\", \"array\"]' [APPLIANCE_ID=appliance_id]"
+    puts " Summary                            - Usage: rake config:summary"
+    puts " Create zone                        - Usage: rake config:create_zone NAME=zone_name DESCRIPTION=zone_description"
+    puts " Modify zone                        - Usage: rake config:modify_zone APPLIANCE_ID=appliance_id ZONE_NAME=zone_name"
+    puts " List appliances                    - Usage: rake config:list_appliances"
+    puts " List zones                         - Usage: rake config:list_zones"
+  end
+
+  desc "List all roles available"
+  task :list_roles do
+    puts role_list.to_json
+  end
+
+  desc "List appliance active roles"
+  task :list_active_roles => [:environment] do
+    if ENV['APPLIANCE_ID'].blank?
+      puts MiqServer.my_server.active_role_names.to_json
+    else
+      puts MiqServer.find(ENV['APPLIANCE_ID']).role.split(",").to_json
+    end
+  end
+
+  desc "Set appliance roles"
+  task :set_roles => [:environment] do
```

Comment:
What's the use case for using this rake task?  Are you trying to automate something?  Can you give more details?